app/src/main/java/net/bible/android/view/activity/navigation/History.kt

```diff
@@ -64,13 +66,28 @@ class History : ListActivityBase() {
      */
     protected fun createAdapter(): ListAdapter {
         mHistoryItemList = historyManager.history
-        val historyTextList: MutableList<CharSequence> = ArrayList()
-        for (item in mHistoryItemList!!) {
-            historyTextList.add(item.description)
-        }
-        return ArrayAdapter(this,
+        return object:ArrayAdapter<HistoryItem>(this,
             LIST_ITEM_TYPE,
-            historyTextList)
+            android.R.id.text1,mHistoryItemList!!){
+            override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
+
+                val view = super.getView(position, convertView, parent)
+                val text1 = view.findViewById(android.R.id.text1) as TextView
+                val text2 = view.findViewById(android.R.id.text2) as TextView
+                text1.text = mHistoryItemList!![position].description.toString()
+
+                val sDt = android.text.format.DateFormat.format("yyyy-MM-dd HH:mm", mHistoryItemList!![position].createdAt).toString()
```

Comment:
Would look better if you import android.text.format.DateFormat. Maybe sDt -> formattedDate (longer and more descriptive variable name)